src/tests/factories/network-connection-factory.ts

```diff
@@ -0,0 +1,16 @@
+import { NetworkConnection } from "andculturecode-javascript-core";
+import { Factory } from "rosie";
+
+// -----------------------------------------------------------------------------------------
+// #region Factory
+// -----------------------------------------------------------------------------------------
+
+export const networkConnectionFactory = Factory.define<
+    NetworkConnection & { id: number }
```

Comment:
What's this `id` used for? Not seeing it referenced anywhere

Reply:
It was a "dumb" effort to make sure it was a unique object.  😄